### ISSUE TITLE
about the inference speed

### ISSUE THREAD

**Issue by Danee-wawawa** [ucbdrive/skipnet#11]
Hi,  thank you for your work. I encountered a problem and hope to get your help.
I first used the supervised pretraining (SP) part of SkipNet to train the model, and got a new model, but when I used PYTorch for the forward speed test, I found that the speed did not decrease. I tried to increase the https:/ /github.com/ucbdrive/skipnet/blob/master/imagenet/models.py#L278, the parameter value here is 0.5 to 0.9. It is found that the accuracy of the model on the test set is indeed reduced a lot, but the forward speed is still not improved, almost the same. Do I need to modify the model in inference?
I am very confused about where the problem is, I hope to get your help, thank you very much.

**Comment by wangqiim** [comment]
Yes, in this code it can't improve speed, because it excatly excute all conv(need be skip by test if times 0), then times 1 or 0, it represent excute or skip. I think you should modify it. I think a easy way to modfiy is to adjust bachsize to 1, then modify code(if prob = 0, don't excute conv).

**Comment by akinsanyaayomide** [comment]
Hello @Danee-wawawa . Have you been able to resolve this problem?
